lib/game_team.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+# GameTeam
+class GameTeam
+  attr_reader :game_id,
```

Comment:
As mentioned in the fixture files, we can possibly remove some of these fields after we determine if they aren't needed in the methods.